engine-standalone-tracing/src/types/call_tracer.rs

```diff
@@ -68,14 +74,17 @@ impl CallTracer {
 
         let mut frame = self.call_stack.pop().unwrap();
         match error {
-            None => frame.output = output,
-            Some(error) => {
-                frame.error = Some(format!("{:?}", error));
+            None => {
                 match frame.call_type {
-                    CallType::Create | CallType::Create2 => frame.to = None,
```

Comment:
I've removed this logic because it was incorrect. Even when there is an error in the deploy execution the `to` address that would have been created is still reported. For example, see this [transaction on Etherscan](https://goerli.etherscan.io/tx/0xdeff004bc889227cd416f39014d352421e8a18ada67123ad295f80269fcc9a2f).